docusaurus.config.js

```diff
@@ -102,6 +102,18 @@ module.exports = {
           label: `Home`,
           position: 'left'
         },
+        {
+          type: 'docSidebar',
+          position: 'left',
+          sidebarId: 'docs',
+          label: 'Documentation',
+        },
+        {
+          type: 'docSidebar',
+          position: 'right',
```

Comment:
The right space is currently reserved for external links (github, slack, ...) while the left is for content :)

```suggestion
          position: 'left',
```

Reply:
Done!